src/main.rs

```diff
@@ -75,6 +76,10 @@ fn app() -> Result {
     cmds.add("?tags get-all", tags::get_all);
     cmds.add("?tags help", tags::help);
 
+    // crates.io
+    cmds.add("?crate", crates::help);
+    cmds.add("?crate query...", crates::search);
```

Comment:
What do you think about `?crate help` and `?crate search query...`?

Reply:
+1, it will make it more open to future commands

Reply:
The current way also ends up searching for the `help` crate when `?crate help` is used

Reply:
i like `?crate help` showing a help message instead of searching for a crate, but i would prefer if the commonly used commands have a concise name - (?tag x over ?tags get x). from testing, having both `?crate help` and `?crate query...` works fine